lib/lm_docstache/hide_custom_tags.rb

```diff
@@ -0,0 +1,65 @@
+module LMDocstache
+  class HideCustomTags
+    HIDE_BACKGROUND_COLOR = 'FFFFFF'
+
+    attr_reader :document, :hide_custom_tags
+    def initialize(document:, hide_custom_tags: [])
+      @document = document
+      @hide_custom_tags = hide_custom_tags
+    end
+
+    def hide_custom_tags!
```

Comment:
It'd be nice to provide through comments some context what this method actually does and what it's needed for. Or make it so for the whole class instead to only this method.